docs/Earn/bridge/bridge-introduction.md

```diff
@@ -1,6 +1,59 @@
 ---
-title: Introduction
+title: Ankr Bridge
 id: bridge-introduction
 ---
+# Ankr Bridge
+
+## What is Ankr Bridge?
+
+Ankr Bridge is an Ankr product for bridging assets between networks. 
+
+It allows you to transfer your Ankr Liquid Staking tokens, obtained when staking your original assets, to a different network where they remain a valid stake and continue to accumulate rewards.
+
+## Why Ankr Bridge?
+
+### User benefits
+
+Having staked your funds in Ankr Earn, you may find that you need them on the spot for some exciting opportunity you don't want to miss.
+However, unstaking may apply different release time for different tokens, and you may miss this opportunity.
+
+In this case, you can transfer your Ankr Liquid Staking tokens to any networks via Ankr Bridge.
+You can then use other Ankr Earn tools to earn more crypto by staking, arbitrage trading, liquidity mining, yield farming, and much more.
+As a part of that, you can trade the native tokens for your Liquid Staking tokens on a DEX, via Ankr DeFi on ankr.com (DEX fees may apply). 
+
+### Ankr benefits
+
+Why did we need our own bridge instead of using existing solutions? 
+
+* Almost all bridges are centralized and we’d like to manage our own solution.
+
+* Almost all bridges support only well-known ERC20 tokens, like the TOP30, and they don’t allow to bridge any ERC-20 compatible asset.
+
+* They don’t support NFT’s standards, such as ERC721.
```

Comment:
NFT (remove 's). 
Add "whereas Ankr Bridge does" at the end of sentence

Reply:
Suggest presenting these features in the positive about Ankr Bridge. i.e. Ankr Bridge uniquely supports NFT standards such as ERC721. unlike other bridges.

Reply:
removed s

not gonna add, as it's miles away from us. 
it's a ground for the Bridge, but it doesn't support NFT now and no plans are made for the near future.